src/main/java/uk/gov/hmcts/darts/common/entity/AnnotationDocumentEntity.java

```diff
@@ -91,6 +91,12 @@ public class AnnotationDocumentEntity extends ModifiedBaseEntity
     @Column(name = "ret_conf_reason")
     private String retConfReason;
 
+    @Column(name = "subcontent_object_id")
+    private String subcontentObjectId;
+
+    @Column(name = "subcontent_position")
+    private Integer subcontentPosition;
```

Comment:
Validation (See previous comment)?

Reply:
See earlier response to similar comment.